### PR TITLE
Testing the docker periodic job

### PR DESCRIPTION
**Pull request by yashsingh74**
This PR is created to test the failed job.

### CODE REVIEW

File: config/jobs/tce/periodics.yaml [hunk 1]
```diff
@@ -112,3 +112,31 @@ periodics:
...
+    containers:
+    - image: public.ecr.aws/t0q8k6g2/vagator/test-image:latest
+      command:
+      - /usr/local/bin/run.sh
+      args:
+      - vmware-tanzu
+      - community-edition
+      - docker pull hello-worldd
+      securityContext:
+        privileged: true
```
Comment (navidshaikh) [inline config/jobs/tce/periodics.yaml:142]: no new line at the end of the file

Comment (yashsingh74) [reply config/jobs/tce/periodics.yaml:142]: This is for testing the sample job, I will remove the job in the later PR.

File: config/jobs/tce/periodics.yaml [hunk 2]
```diff
@@ -112,3 +112,31 @@ periodics:
       nameservers:
         - 1.1.1.1
         - 1.0.0.1
+
+- name: periodic-testing-docker
+  cluster: test-jobs
+  decorate: true
+  # interval:  24h
+  cron: 0/5 * * * *
```
Comment (navidshaikh) [inline config/jobs/tce/periodics.yaml:120]: this needs a todo to update else it'd be too often

**Review by navidshaikh — APPROVED** [review]
/lgtm

**Pull request merged**